python/mxnet/symbol/numpy/_symbol.py

```diff
@@ -31,13 +31,107 @@
 
 __all__ = ['zeros', 'ones', 'maximum', 'minimum', 'stack', 'concatenate', 'arange', 'argmax',
            'clip', 'add', 'subtract', 'multiply', 'divide', 'mod', 'power', 'split', 'swapaxes',
-           'expand_dims', 'tile', 'linspace', 'sin', 'cos', 'sinh', 'cosh', 'log10', 'sqrt']
+           'expand_dims', 'tile', 'linspace', 'sin', 'cos', 'sinh', 'cosh', 'log10', 'sqrt', 
+           'absolute', 'cbrt', 'arccos']
 
+@set_module('mxnet.symbol.numpy')
+def absolute(x, out=None, **kwargs):
+    r"""
+    Calculate the absolute value element-wise.
+    np.abs is a shorthand for this function.
+
+    Parameters
+    ----------
+    x : _Symbol
+    Input array.
+
+    out : _Symbol, optional
+    A location into which the result is stored. If provided, it must have a shape 
```

Comment:
According to the template, all "out" params in _symbol.py had better be commented with
"Dummy parameter to keep the consistency with the ndarray counterpart."

Reply:
Thx. Fixed.